controllers/kubevirtcluster_controller.go

```diff
@@ -84,8 +84,6 @@ func (r *KubevirtClusterReconciler) Reconcile(goctx gocontext.Context, req ctrl.
 		return ctrl.Result{}, nil
 	}
 
-	log = log.WithValues("cluster", cluster.Name)
```

Comment:
why was this removed?

Reply:
It causes the linter to fail, because this variable is declared and never used(I put this deletion in a separate commit)

Reply:
oh! alrighty